src/app/ApplicationTemplate.UWP/Views/Styles/Application/Brushes.xaml

```diff
@@ -42,25 +42,25 @@
 					 Color="{ThemeResource MaterialOnPrimaryColor}" />
 	<SolidColorBrush x:Key="MaterialOnPrimaryLowBrush"
```

Comment:
We should not redefine existing Material brushes. We only need to define the color palette. The brushes are updated automatically.